apps/web-giddh/src/app/new-inventory/component/custom-price/advance-list-items-popup/advance-list-items-popup.component.scss

```diff
@@ -0,0 +1,215 @@
+:host {
+    .search-wrapper {
+        padding: 1.25rem 2rem 1.25rem 4.2rem;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        background-color: var(--color-popup-background);
+        flex-wrap: wrap !important;
+        width: 100% !important;
+        max-width: 580px;
+        .search-element {
+            outline: none;
+            border: none;
+            font-size: var(--font-size-15);
+            font-weight: normal;
+            line-height: initial;
+            width: 95%;
+            background-color: var(--color-popup-background);
+        }
+        .icon-search-icon {
+            position: absolute;
+            left: 0;
+            right: 0;
+            width: 4.2rem;
+            text-align: center;
+            color: var(--color-abbey);
+        }
+    }
+    .vR {
```

Comment:
don't use capital letter in css. Change it to .v-r or something else. Fix this issue in all places.

Reply:
Ok sir this old class define in cmd+k component but I correct it in my component